app/config/parameters.yml.dist

```diff
@@ -10,13 +9,13 @@ parameters:
 
     mailer_transport: '%env(SYLIUS_MAILER_TRANSPORT)%'
     mailer_host: '%env(SYLIUS_MAILER_HOST)%'
+    mailer_port: '%env(SYLIUS_MAILER_PORT)%'
```

Comment:
Could we extract this change to another PR?

Reply:
But we need it here...